app/atomic_api.py

```diff
@@ -0,0 +1,28 @@
+import json
+import logging
+from aiohttp import web
+from aiohttp_jinja2 import template
+
+from app.utility.base_service import BaseService
+from app.service.auth_svc import for_all_public_methods, check_authorization
+from plugins.atomic.app.atomic_svc import AtomicService
+
+
+@for_all_public_methods(check_authorization)
+class AtomicAPI(BaseService):
```

Comment:
Should the api class be extending `BaseService`? It's not a service so this feels off to me. FWIW, the skeleton plugin doesn't do this.

Reply:
Also should this be `AtomicGui` since it's not implementing an API?